contrib/world_builder/include/world_builder/types/int.h

```diff
@@ -60,8 +60,8 @@ namespace WorldBuilder
                           const std::string &name,
                           const std::string &documentation) const override final;
 
-        unsigned int value {0};
-        unsigned int default_value;
+        int value {0};
+        int default_value;
```

Comment:
Why this change? What doe these values mean?

Reply:
This was a bug, it should always have been int, because this is a type which stores a int. The values store the actual and the default value in the type.